frame.rkt

```diff
@@ -723,7 +717,19 @@ GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>."
-                               ivy-version)))]))
+                       ivy-version)))
+
+(cond
+  [macosx?
+   (application-about-handler show-about-dialog)]
+  [else
+   (define ivy-menu-bar-help-about
```

Comment:
Since this is a context local to the `cond`, we're never going to be able to reference it at a higher context, so it might be better to wrap the `new menu-item%` inside a `void`. This way you don't have to have the dangling `#f` and the `menu-item%` will be created anyway.

Reply:
Done. I actually thought about this; I'm not sure why I decided against it.